getting_started.ipynb

```diff
@@ -144,42 +136,36 @@
    },
    "source": [
     "## Step 3: Visualize data\n",
-    "First some additional imports for plotting."
+    "Preview the data before doing further calculations. This step may take a few minutes to compute, because the data is only loaded \"lazily\" until you output it (in visualize or export). "
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
-   "id": "62695959-ff1a-4b59-835d-01454f5fbf42",
+   "id": "df5326c6-01fb-44f6-a20c-ad0fd316a2d4",
    "metadata": {},
    "outputs": [],
    "source": [
-    "import hvplot.xarray\n",
-    "import cartopy.crs as ccrs"
+    "data = data_to_use.compute()\n",
```

Comment:
I would recommend moving `data_to_use.compute()` into the previous section "Step 2: Retrieve Data". Because it is more about data retrieval, not visualization. By calling compute(), we load the data into memory. I also don't like how the dataset is renamed from `data_to_use` to just `data`; I think it's a bit confusing, because it's the same data, just loaded into memory. In step two, I would recommend adding a cell and a description: 


"Next, load the data into memory. This allows the previous steps to run faster"
`data_to_use = data_to_use.compute()`